gcs-action/.github/workflows/test.yml

```diff
@@ -0,0 +1,71 @@
+name: 'build-test'
+on: # rebuild any PRs and main branch changes
+  pull_request:
```

Comment:
You can just use: `on: [push, pull_request]`

Reply:
Done